project-base/tests/FrontendApiBundle/Functional/Advert/GetAdvertsTest.php

```diff
@@ -0,0 +1,132 @@
+                'link' => null,
```

Comment:
image is the same for all three adverts, right? 
I'm thinking about extract into the separate method (getAdvertCode), what do you think?

Reply:
there remained only one occurrence